crossplane.yaml

```diff
@@ -39,4 +39,4 @@ spec:
     - provider: xpkg.upbound.io/upbound/provider-aws
       version: ">=v0.14.0-0"
     - provider: xpkg.upbound.io/crossplane/provider-helm
-      version: ">=v0.10.0-0"
+      version: ">=v0.11.0-0"
```

Comment:
Same comment as https://github.com/upbound/platform-ref-azure/pull/11#discussion_r979064623 😊

Reply:
provider-helm is [published](https://marketplace.upbound.io/providers/crossplane/provider-helm/v0.11.1) and I updated both versions!